monkeyhex.py

```diff
@@ -2,14 +2,22 @@
 from future.utils import iteritems
 from past.builtins import long
 
+HEX_THRESHOLD = 10**17 # since that hex is more succinct ...
+alwaysHex = True
```

Comment:
- we write our variables in snake_case around here
- I would rather the toggle for enabling the feature be `HEX_THRESHOLD = None`

Reply:
1. OK
2. the problem with it that a user would have to either to calculate the threshold himself, or we would have to store it in a separate variable and then a user would have to assign the threshold to it. Calculating himself is of course not an option: this is a bad UX. Setting `monkeyhex.HEX_THRESHOLD = monkeyhex.DEFAULT_HEX_THRESHOLD` is also looks like poor UX: in order to use the lib a user has to remember 2 variable names instead of 1.